recipes/libcst/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "libcst" %}
+{% set version = "0.3.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 449bf67813e4ced937edb9811355e2734ac709b4f36416889c6e169a03b96362
+
+build:
+  number: 0
+  skip: True  # [py < 36]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - dataclasses   # [py < 37]
+    - pip
+    - python
+    - pyyaml >=5.2
+    - typing_extensions >=3.7.2
+    - typing_inspect >=0.4.0
+  run:
+    - dataclasses   # [py < 37]
+    - python
+    - pyyaml >=5.2
+    - typing_extensions >=3.7.2
+    - typing_inspect >=0.4.0
+
+test:
+  imports:
+    - libcst
+    - libcst._nodes
+    - libcst._nodes.tests
+    - libcst._parser
+    - libcst._parser.conversions
+    - libcst._parser.parso
+    - libcst._parser.parso.pgen2
+    - libcst._parser.parso.python
+    - libcst._parser.parso.tests
+    - libcst._parser.tests
+    - libcst._parser.types
+    - libcst._parser.types.tests
+    - libcst.codegen
+    - libcst.codegen.tests
+    - libcst.codemod
+    - libcst.codemod.commands
+    - libcst.codemod.commands.tests
+    - libcst.codemod.tests
+    - libcst.codemod.visitors
+    - libcst.codemod.visitors.tests
+    - libcst.helpers
+    - libcst.helpers.tests
+    - libcst.matchers
+    - libcst.matchers.tests
+    - libcst.metadata
+    - libcst.metadata.tests
+    - libcst.testing
+    - libcst.tests
+
+about:
+  home: "https://github.com/Instagram/LibCST"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "A concrete syntax tree with AST-like properties for Python 3.5, 3.6, 3.7 and 3.8 programs."
+  doc_url: https://libcst.readthedocs.io/
+  dev_url: https://github.com/Instagram/LibCST
+
+extra:
+  recipe-maintainers:
+    - nwani
```

Comment:
hm...

Reply:
but who is @nwani?

Reply:
My alter ego 🤫🤫🤫🤫🤫

Reply:
Now that I think about it ... someone stole my alter ego name on Github 😜

Reply:
cool, i've had a few: just didn't want any surprises!